metadata.rb

```diff
@@ -4,17 +4,15 @@
 license          'Apache-2.0'
 description      'Installs/Configures Splunk Servers and Forwarders'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.21.1'
+version          '2.21.2'
 
 source_url       'https://github.com/cerner/cerner_splunk'
 issues_url       'https://github.com/cerner/cerner_splunk/issues'
 
-chef_version     '~> 12.4' if respond_to?(:chef_version)
+chef_version     '>= 12.7', '< 15' if respond_to?(:chef_version)
 
 depends          'chef-vault', '~> 3.0'
-depends          'ulimit', '~> 0.3'
-depends          'xml', '~> 1.2'
-
```

Comment:
Style: keep this separation between cookbook dependencies and supported platform metadata....